src/ray/gcs/gcs_server/gcs_actor_manager.h

```diff
@@ -339,6 +339,8 @@ class GcsActorManager : public rpc::ActorInfoHandler {
 
   std::string DebugString() const;
 
+  bool &SchedulePendingActorsPosted();
```

Comment:
Don't use it this wary. Use setter/getter instead.

Reply:
Fixed